inc/cleantalk-ajax.php

```diff
@@ -401,7 +401,8 @@ function ct_ajax_hook($message_obj = null)
          (Post::get('action') === 'uael_login_form_submit') || // skip Ultimate Addons for Elementor login
          (Post::get('action') === 'my_custom_login_validate') || // skip Ultimate Addons for Elementor login validate
          (Post::get('action') === 'wpforms_restricted_email') || // skip WPForm validate
-         (Post::get('action') === 'fluentcrm_unsubscribe_ajax') // skip fluentcrm unsubscribe
+         (Post::get('action') === 'fluentcrm_unsubscribe_ajax') ||// skip fluentcrm unsubscribe
+         (Post::get('action') === 'klc_generate_coupon') // skip the client's plugin for issuing coupons
```

Comment:
Please, extend condition via apbct_is_plugin_active and move this to the apbct_is_skip_request()
https://github.com/CleanTalk/wordpress-antispam/blob/eff5b82ed11b5a9411ce8d925cbd4bed1f5deb81/inc/cleantalk-pluggable.php#L534

Reply:
Solved https://github.com/CleanTalk/wordpress-antispam/commit/bb569649b5ad9a5cac68a350f1041681b633c213